test/configCases/target/node-dynamic-import/warnings.js

```diff
@@ -0,0 +1,3 @@
+module.exports = [
+	[/System\.import\(\) is deprecated/]
+];
```

Comment:
Add additional config testcases for `system: true`,`system: undefined` and `system: false`